src/components/Header/Navigation.jsx

```diff
@@ -0,0 +1,39 @@
+import { useAuth } from "../../hooks";
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+
+const Link = styled(NavLink)`
```

Comment:
I suppose It`s temprorary and if you will have chosen styled components they will be separate from markup